tutorials/context-dependent-tool-tutorial.md

```diff
@@ -1,11 +1,17 @@
+---
+layout: doc
+title: Intermediate Tool Tutorial: Context-dependent Tools
+permalink: /docs/tutorials/context-dependent-tools
+---
+
 ## Intermediate Tool Tutorial: Context-dependent Tools
 
 Pre-req: [Creating a Simple Tool](../develop/spatial-tools/tutorial)
 
 Now that you know how to create a tool and interact with some basic RealityInterface APIs,
 we're going to explore some more interesting things you can do with tools. This tutorial gets a
-bit complicated, but as always please [use the forum](forum.spatialtoolbox.vuforia.com) if you
-have any questions.
+bit complicated, but as always please [use the forum](https://forum.spatialtoolbox.vuforia.com) if
+you have any questions.
 
 ##### Setting Up
```

Comment:
I think this might need to be ### instead. Sorry for not catching this earlier!

Reply:
oops I merged too soon. I can fix in a follow-up